### DISCUSSION TITLE
Minimum version for pykeepass?

### DISCUSSION THREAD

**Discussion by adbrown101** [roddhjav/pass-import#153] [Q&A] [answered]
Hi, I am trying to update the  [pass-import package](https://github.com/void-linux/void-packages/pull/32091) to 3.2 for Void Linux and I am finding that the test `tests/exports/test_keepass.py` fails with our version of pykeepass (3.2), but passes if I update to 4.0.1. Is there a minimum version of pykeepass that works with version 3.3 of pass-import?

**Comment by roddhjav** [comment, marked as answer]
Yes, there is, currently you need at least pykeepass 4.0.0

**Comment by glasseyes** [comment]
I think pass-import 3.4 requires pykeepass  4.0.3 and later

From pykeepass changelog
4.0.3 - 2022-06-21
added otp support

reading kdbx fails at formats/kdbx.py:72 if otp support is missing

**Reply by adbrown101** [reply]
Thanks for the update we are on pass-import 3.4 and pykeepass 4.0.3
